content/concepts/peer-routing.md

```diff
@@ -3,6 +3,80 @@ title: Peer Routing
 weight: 5
 ---
 
-This article is coming soon!
+Libp2p primarily uses Kademlia DHT for peer discovery. Let's dive a little bit into how Kademlia works.
```

Comment:
It is one mechanism among many. E.g. ethereum 2 is not using it at all.

Reply:
fixed in new commit, but still not sure if it sounds good, since I don't know yet many other ways to discover peers 🙃